frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -42,30 +120,137 @@ function renderAnalysis() {
+  return html`
+    <btrix-table
+      class="mx-2"
+      style="grid-template-columns: ${[
+        "[clickable-start] auto",
+        "12rem",
+        "12rem [clickable-end]",
+      ].join(" ")}"
+    >
+      <btrix-table-head>
+        <btrix-table-header-cell>${msg("Page")}</btrix-table-header-cell>
+        <btrix-table-header-cell>${msg("Approval")}</btrix-table-header-cell>
+        <btrix-table-header-cell>${msg("Comments")}</btrix-table-header-cell>
+      </btrix-table-head>
+      <btrix-table-body class="rounded border">
+        ${component.pages?.items.map(
+          (page, idx) => html`
+            <btrix-table-row
+              class="${idx > 0
+                ? "border-t"
+                : ""} cursor-pointer select-none transition-colors focus-within:bg-neutral-50 hover:bg-neutral-50"
+            >
+              <btrix-table-cell
+                class="block overflow-hidden"
+                rowClickTarget="a"
+              >
+                <a
+                  class="truncate text-sm font-semibold"
+                  href=${`${
+                    component.navigate.orgBasePath
+                  }/items/${component.itemType}/${component.crawlId}/review/screenshots?qaRunId=${
+                    component.qaRunId || ""
+                  }&itemPageId=${page.id}`}
+                  @click=${component.navigate.link}
+                  >${page.title}</a
+                >
+                <div class="truncate text-xs leading-4 text-neutral-600">
+                  ${page.url}
+                </div>
+              </btrix-table-cell>
+              <btrix-table-cell>${pageReviewStatus(page)}</btrix-table-cell>
+              <btrix-table-cell
+                >${page.notes?.length
+                  ? statusWithIcon(
+                      commentIcon,
+                      page.notes.length === 1
+                        ? msg(str`1 comment`)
+                        : msg(
+                            str`${page.notes.length.toLocaleString()} comments`,
+                          ),
+                    )
+                  : html`<span class="text-neutral-400"
+                      >${msg("None")}</span
+                    >`}</btrix-table-cell
+              >
+            </btrix-table-row>
+          `,
+        )}
+      </btrix-table-body>
+    </btrix-table>
+    ${when(component.pages, (pages) =>
+      pages.total > pages.pageSize
+        ? html`
+            <footer class="mt-3 flex justify-center">
+              <btrix-pagination
+                page=${pages.page}
+                size=${pages.pageSize}
+                totalCount=${pages.total}
+                @page-change=${(e: PageChangeEvent) => {
+                  component.fetchPages({
+                    page: e.detail.page,
+                  });
+                }}
+              ></btrix-pagination>
+            </footer>
+          `
+        : nothing,
+    )}
+  `;
+}
+
+/**
+ * TODO convert to lit component
```

Comment:
@emma-sg probably should convert to new lit component, didn't want to conflict with the base branch so could do it after it's merged in

Reply:
Sure, will do!